src/ch01-02-hello-hot-reloading.md

```diff
@@ -0,0 +1,109 @@
+# Hello, Hot Reloading!
+
+Mun distinguishes itself from other languages by its inherent hot reloading capabilities. The
+following example illustrates how you can create a hot reloadable application by extending the
+[Hello, fibonacci?](ch01-01-hello-fibonacci.md) example.
+
+Filename: hello_fibonacci.mun
+
+```mun
+fn fibonacci5():int {
+    fibonacci(5)
+}
+
+fn fibonacci(n:int):int {
+    if n <= 1 {
+        n
+    } else {
+        fibonacci(n-1) + fibonacci(n-2)
+    }
+}
+```
+
+<span class="caption">Listing 1-2: A function that calculates a fibonacci number</span>
+
+Apart from running Mun libraries from the command-line interface, a common use case is
+embedding them in other programming languages.
+
+## Mun embedded in C++
+
+Mun [exposes](github.com/mun-lang/runtime-ffi) a C API and complementary c++ bindings for the Mun
+Runtime. Listing 1-3 shows a C++ application that constructs a Mun Runtime for the
+`hello_fibonacci` library and continuously invokes the `fibonacci5` function and outputs its result.
+
+Filename: main.cc
+
+```cpp
```

Comment:
Hightlighting of cpp doesnt seem to work..

Reply:
Fixed. It was missing a closing **"**